packages/gql/src/pool-snapshots/snapshots.ts

```diff
@@ -0,0 +1,12 @@
+import { gql } from "graphql-tag";
+
+export const PoolSnapshots = gql`
+  query PoolSnapshot($pool: String, $timestamp: Int) {
+    poolSnapshots(where: { pool: $pool, timestamp_gte: $timestamp }) {
```

Comment:
would also need to be sorted by timestamp and/or return the timestamp in the response, no?